Prototyping/Domains/Halloween/ZoneStatueJumpScare/zoneStatueJumpScare.js

```diff
@@ -0,0 +1,348 @@
+//
+// zoneStatueJumpScare.js
+// 
+// Created by Robin Wilson on 09/20/2018
+// Copyright High Fidelity 2018
+//
+// Put script on a zone or collisionless entity and a scary statue follows you.
+//
+// Licensed under the Apache 2.0 License
+// See accompanying license file or http://apache.org/
+//
+// Whisper Sound: https://freesound.org/people/DRFX/sounds/350763/ by DRFX
+
+(function () {
+
+    var Sound = Script.require("http://hifi-content.s3-us-west-1.amazonaws.com/robin/dev/utils/Sound.js?v" + Math.random());
+
+    var overlayID;
+    var entityID;
+
+    var endCondition = false;
+
+    var startPosition;
+    var isRunning = false;
+
+    var distances = [8, 6, 2, 1.5, 1];
+    var count = 0;
+
+    var MIN_TIME_INVISIBLE = 100;
+    var MAX_TIME_INVISIBLE = 2000;
+    
+    var MIN_TIME_VISIBLE = 500;
+    var MAX_TIME_VISIBLE = 1500;
+    
+    var MIN_WAIT_TIME_JUMPSCARE = 5000;
+    var MAX_WAIT_TIME_JUMPSCARE = 6000;
+    
+    var TIME_FINAL_MOVE_VISIBLE = 2000;
+    var TIME_JUMPSCARE_VISIBLE = 1500;
+
+    var SOUND_WHISPER_URL = "http://hifi-content.s3-us-west-1.amazonaws.com/robin/dev/domains/halloween/flashingLight/whisper1.wav";
```

Comment:
In the future, these should be relative

Reply:
Should I change them to be relative for the PR?